### PR TITLE
Add Option to add FloatingActionButton to BackdrawScaffold

### PR DESCRIPTION
**Pull request by felixwortmann**

### CODE REVIEW

**Comment by WieFel** [comment]
Ok for me!
@daadu I don't know whether we should already include the version update and the updated changelog?
I let you handle that...

**Comment by daadu** [comment]
@felixwortmann @WieFel version bump and changelog should not be added in PR.

I have resolved merge conflicts.

LGTM. What about you @WieFel

**Comment by WieFel** [comment]
If we merge #23 before this one then the `appBarBackgroundColor` property could be removed, as it is already handled by `BackdropAppBar`.
Therefore I was thinking of implementing the app bar before accepting this pull request ;)

**Comment by daadu** [comment]
@felixwortmann Please do the agreed changes.

**Comment by daadu** [comment]
@felixwortmann One more change:
- refactor `backPanelBackgroundColor` -> `backLayerBackgroundColor` to be consistent with nomenclature.

**Comment by daadu** [comment]
@felixwortmann Please confirm if you will be working on the changes.

**Comment by felixwortmann** [comment]
@daadu I will implement the changes later today or this week

**Comment by felixwortmann** [comment]
@daadu I think this should be it. Can you take a look please?

File: lib/backdrop.dart [hunk 1]
```diff
@@ -141,19 +141,23 @@ class BackdropScaffold extends StatefulWidget {
   /// Background [Color] for the BackPanel
```
Comment (daadu) [inline lib/backdrop.dart:141]:
```suggestion
  /// Background [Color] for the back layer.
```

**Comment by daadu** [comment]
LGTM. @WieFel What do you think?

**Comment by WieFel** [comment]
Approved!

**Comment by daadu** [comment]
Congrats @felixwortmann your code have landed to `v0.3.1` on pub.dev. Also check yourself in [Contributor's Section on README](https://github.com/fluttercommunity/backdrop#contributors-)

**Pull request merged**